.github/workflows/tests_linux_cpp.yml

```diff
@@ -146,7 +147,9 @@ jobs:
               -DSCIPY_OPENBLAS=$(python -c "import scipy_openblas32; print(scipy_openblas32.get_lib_dir())") \
               -DBUILD_TESTS=ON \
               -DCMAKE_CXX_COMPILER=$(which g++-$GCC_VERSION) \
-              -DENABLE_COVERAGE=ON
+              -DENABLE_COVERAGE=ON \
+              -DCATALYST_GIT_TAG="device-interface-rework"
+            # TODO: remove CATALYST_GIT_TAG after release
```

Comment:
Same here. We need to have an explicit version, as this knowledge can be lost with time.